File: .sdk/tm/csharp/test/ExistsTest.cs
// Solardemo SDK exists test.

using Xunit;

using SolardemoSdk;

namespace SolardemoSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = SolardemoSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: .sdk/tm/kotlin/test/ExistsTest.kt
package KOTLINPACKAGE.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import KOTLINPACKAGE.core.SolardemoSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = SolardemoSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
